test/utils.test.js

```diff
@@ -196,4 +202,14 @@ describe('utils', () => {
             expect(isPlainObject(Symbol('test'))).toBe(false);
         });
     });
+
+    describe('zipObject', () => {
+        it('should return the zipped object', () => {
+            expect(zipObject(['a', 'b'], [1, 2])).toStrictEqual({ a: 1, b: 2 });
```

Comment:
Need more tests to cover possible variants, like 
+`(['a', 'b', 'c'], [1, 2])`
+`(['a'], [1, 2])`

Reply:
Feel free to add them but I'm not sure if they'll work. What did the lodash zipObject return in these cases?

Reply:
@Kreeg did you confirm what lodash is returning with unbalanced arrays?

Reply:
So, the current behavior is the same, right?

Reply:
@XhmikosR yes